common/dtd/ldmlSupplemental.dtd

```diff
@@ -1278,7 +1278,7 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 <!ATTLIST coverageLevel inLanguage CDATA #IMPLIED >
     <!--@MATCH:any-->
 <!ATTLIST coverageLevel inScript CDATA #IMPLIED >
-    <!--@MATCH:validity/script-->
+    <!--@MATCH:any-->
```

Comment:
This is so we can use coverage variables for inScript, as we do for inLanguage for example.

Reply:
ideally `@MATCH:validity/script-with-variables` or something

Larger question is whether coverage level belongs in "supplemental data" when, like casing, it's primarily inward tool-focused data.  Perhaps we should split some of these out -- and give them their own DTD.

Reply:
Filed https://unicode-org.atlassian.net/browse/CLDR-17618